.github/workflows/build.yml

```diff
@@ -0,0 +1,48 @@
+name: "Build And Publish"
+on: push
```

Comment:
Jos tässä on joku publish mukana, niin olis ehkä hyvä että tää triggeröityis vaan tietyissä brancheissa:

```
on:
  push:
    branches:
      - master
```

Ja sit jos halutaan kutenkin ajaa joku build esim. aina kun joku tekee pull requestin, niin sit pitäis tehdä vielä erillinen github action missä on vaan build ja siinä triggerinä on `on: pull_request`.

Reply:
Joka pushista tulee draft-release, joka näkyy vain vihityille. Eli niitä buildeja ei mun mielestä tartte säästellä.

Mutta tekishän se eetvarttia, että buildi tehtäis myös pull_requestista. Lisään ton `pull_request` eventin.

Reply:
Niin kait toi push mihin tahansa branchiin vaatii sen että on contributor oikeudet repoon.

Ton pull requestin jättäisin pois publish actionista, kun se tarkottaa että kuka tahansa ulkopuolinen voi triggeröidä ton buildin ja laittaa sisään mitä tahansa